charts/openstack-actions-runner/templates/deployment.yaml

```diff
@@ -33,10 +33,14 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           envFrom:
-          - configMapRef:
-              name: {{ .Release.Name }}-config
-          - secretRef:
-              name: {{ .Release.Name }}-auth
+            - configMapRef:
+                name: {{ .Release.Name }}-config
+            - secretRef:
+                name: {{ .Release.Name }}-auth
+          volumeMounts:
+            - name: {{ .Release.Name }}-settings
+              mountPath: /app
```

Comment:
is the default path to the settings app on the file `/app/settings.yaml` ? if so then LGTM